skyportal/handlers/api/candidate.py

```diff
@@ -0,0 +1,280 @@
+import datetime
+import arrow
+from sqlalchemy.orm import joinedload
+from marshmallow.exceptions import ValidationError
+from baselayer.app.access import auth_or_token, permissions
+from ..base import BaseHandler
+from ...models import (
+    DBSession,
+    Photometry,
+    Thumbnail,
+    Instrument,
+    Group,
+    Source,
+    GroupSource,
+    GroupCandidate,
+)
+
+
+CANDIDATES_PER_PAGE = 25
+
+
+class CandidateHandler(BaseHandler):
+    @auth_or_token
+    def get(self, candidate_id=None):
+        if candidate_id is not None:
+            c = Source.get_if_owned_by(
+                candidate_id,
+                self.current_user,
+                groups_attr="candidate_groups",
+                options=[
+                    joinedload(Source.candidate_comments),
+                    joinedload(Source.candidate_groups),
+                    joinedload(Source.thumbnails)
+                    .joinedload(Thumbnail.photometry)
+                    .joinedload(Photometry.instrument)
+                    .joinedload(Instrument.telescope),
+                ],
+            )
+            if c is None or not c.is_candidate:
```

Comment:
add parentheses for clarity: `(c is None) or (not c.is_candidate)`

Reply:
Done.